### ISSUE TITLE
Aktywacja konta sandbox

### ISSUE THREAD

**Issue by JakubOkrzesik** [allegro/allegro-api#8879]
Prosiłbym o aktywację konta na sandboxie
username: bigbapo
Z góry dziękuję

**Comment by PrzemyslawLukanowski** [comment]
Aktywowałem Twoje konto.

**Comment by WojciechGwzdon** [comment]
Ja również prosze o aktywacje konta na sandboxie
login: wgwizdon

**Comment by PrzemyslawLukanowski** [comment]
@therodzyn Aktywowałem Twoje konto. Jeśli chciałbyś wystawiać i zarządzać ofertami przez API, musisz posiadać konto firmowe - w tym celu możesz zmienić rodzaj konta lub założyć nowe (a obecne, zwykłe, zostawić jako drugie - kupującego).

**Comment by therodzyn** [comment]
@PrzemyslawLukanowski 
Dziękuję. Próbowałem dodać aplikację https://apps.developer.allegro.pl.allegrosandbox.pl/, ale nic się nie działo po kliknięciu przycisku Dodaj (w konsoli przeglądarki zwracało mi błąd, że nie mam włączonego dwustopniowego logowania, mimo, że na sandboxie to jest ponoć wyłączone).
Po zmianie na konto firmowe chyba konieczna jest ponowna aktywacja, ponieważ teraz w ogóle nie mogę zarejestrować aplikacji.

**Comment by PrzemyslawLukanowski** [comment]
@therodzyn Skonfiguruj jeszcze proszę [ustawienia wypłat]([skonfiguruj wypłaty](https://allegro.pl.allegrosandbox.pl/moje-allegro/sprzedaz/ustawienia-wyplat)), abym mógł aktywować to konto.

**Comment by therodzyn** [comment]
@PrzemyslawLukanowski Skonfigurowałem ustawienia wypłat.

**Comment by PrzemyslawLukanowski** [comment]
@therodzyn Aktywowałem konto, sprawdź proszę, czy aktualnie możesz zarejestrować nową aplikację.

**Comment by therodzyn** [comment]
@PrzemyslawLukanowski Mam teraz dostęp do formularza rejestracji aplikacji, ale gdy wprowadzę dane (nazwa, Aplikacja działa w środowisku bez dostępu do przeglądarki lub klawiatury i zaznaczone wszystkie uprawnienia) i klikam ZAREJESTRUJ nic się nie dzieje. Dopiero w konsoli przeglądarki mogę podejrzeć, że idzie request XHR, ale mam odpowiedź 403:
`{
    "errors": [
        {
            "code": "USER_WITHOUT_2FA",
            "message": "Twoje konto nie ma włączonego dwustopniowego uwierzytelniania. Link:https://allegro.pl/pomoc/dla-kupujacych/logowanie-i-haslo/czym-jest-dwustopniowe-logowanie-i-dlaczego-warto-z-niego-korzystac-dykqg9nMKSZ",
            "details": null,
            "path": "userWithout2FA",
            "userMessage": "Twoje konto nie ma włączonego dwustopniowego uwierzytelniania. Link:https://allegro.pl/pomoc/dla-kupujacych/logowanie-i-haslo/czym-jest-dwustopniowe-logowanie-i-dlaczego-warto-z-niego-korzystac-dykqg9nMKSZ"
        }
    ]
}`

Dwustopniowego logowania nie mogę dodać, bo nie dochodzą SMSy.

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za informację, weryfikujemy ten problem, wrócę z odpowiedzią.

**Comment by PrzemyslawLukanowski** [comment]
W międzyczasie możesz włączyć dwustopniowe logowanie Sandboxie - wpisz 12345 do aktywacji, a przy samym logowaniu 123456.

**Comment by therodzyn** [comment]
@PrzemyslawLukanowski Dzięki! Po dodaniu dwustopniowego logowania na podstawie powyższych danych udało mi się zarejestrować aplikację 👍 
Dodałem też nowe konto erpit_kupujacy z którego będę składał zamówienia. Czy ono też wymaga aktywacji?

**Comment by PrzemyslawLukanowski** [comment]
Nie, to konto zostanie automatycznie aktywowane przy pierwszy zakupie :)

**Comment by XDSWXD** [comment]
mam problem z aktywacją konta xdefttestowe, zakup zrobiłem, dwustopniowe logowanie włączone, niestety dalej nie mogę dodać aplikacji, proszę o podpowiedź co jeszcze powinienem zrobić @PrzemyslawLukanowski

**Comment by PrzemyslawLukanowski** [comment]
@XDSWXD Spróbuj proszę teraz, aktywowałem konto po naszej stronie.

**Comment by XDSWXD** [comment]
> @XDSWXD Spróbuj proszę teraz, aktywowałem konto po naszej stronie.

Udało mi się powiązać aplikację, dzięki za pomoc. 
Jesteś w stanie pomóc z aktywowaniem oferty? https://allegro.pl.allegrosandbox.pl/oferta/7775815334

**Comment by PrzemyslawLukanowski** [comment]
Widzę, że oferta jest już aktywna.

**Comment by XDSWXD** [comment]
> Widzę, że oferta jest już aktywna.

Fakt, dzięki za info!

**Comment by therodzyn** [comment]
Dostałem blokadę ochrony marek na sandboxie.
Mogę prosić o odblokowanie? Czy na sandboxie również będę dostawał blokady?

**Comment by PrzemyslawLukanowski** [comment]
@therodzyn Możesz potwierdzić warunki ochrony marek poprzez [ten link](https://salescenter.allegro.com.allegrosandbox.pl/brands-to-protect?bi-s=art)? Jeśli problem nadal będzie występować, prześlij proszę przykładowy zrzut ekranu z błędem.

**Comment by therodzyn** [comment]
@PrzemyslawLukanowski Potwierdziłem, oferta wystawiona. Dzięki!

**Comment by fpagacz-sketch** [comment]
prosze o aktywacje sandbox 
user :  outlet_rowerowy

**Comment by MaciejFrackowiak** [comment]
@fpagacz-sketch Odpowiedziałem w [wątku](https://github.com/allegro/allegro-api/issues/12791#issuecomment-3635968276).
